Examples/Run/Reconstruction/DD4hep/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+add_executable(
+  ActsExampleTruthTracksDD4hep
+  DD4hepRecTruthTracks.cpp)
+
+target_link_libraries(
+	ActsExampleTruthTracksDD4hep
```

Comment:
```suggestion
  ActsExampleTruthTracksDD4hep
```
Whitespace.

Reply:
I don't realize that the format here has been fixed manually.. Fixed.